R/magpie-class.R

```diff
@@ -238,7 +238,7 @@ setClass("magpie", contains = "array", prototype = array(0, c(0, 0, 0)))
   pmatch2 <- ifelse(pmatch == TRUE | pmatch == "left", "[^.]*", "")
 
   if (!is.list(i)) i <- list(i)
-  elems <- 1:dim(x)[dim]
+  elems <- 1:dim(x)[dim] # nolint: seq_linter
```

Comment:
This is probably what you meant with "never touch a running system", but this seems very safe to me...
```suggestion
  elems <- seq_len(dim(x)[dim])
```